src/controllers/adminController.ts

```diff
@@ -1,6 +1,6 @@
-import * as fs from "fs";
+import fs from "fs";
 
-import { Request, Response, NextFunction } from "express";
+import { Request, Response, NextFunction, application } from "express";
```

Comment:
I don't think the `application` import from express is required in the adminController

Reply:
Never mind, this is fixed in https://github.com/unicsmcr/hs_application/pull/59/files#diff-577af97e69e695622bf1b75293c0b458R3